locale/aln/app.po

```diff
@@ -4463,3 +4463,12 @@ msgstr ""
 
 msgid "« Back to search results"
 msgstr ""
+
+#~ msgid "Overdue"
```

Comment:
These shouldn't get committed. See "You may need to manually remove obsolete msgids." on https://github.com/mysociety/alaveteli/wiki/Release-Manager's-checklist